src/components/inputs/TagSelectInput.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <q-select
+    :model-value="modelValue"
+    :options="tags"
+    dense
+    outlined
+    multiple
+    color="accent"
+    options-selected-class="text-accent"
+    class="search-bar-input q-mr-md"
+    @update:model-value="(value) => $emit('update:modelValue', value)"
+  >
+    <template #selected>
+      <div
+        v-if="modelValue.length === 0"
+        class="text-grey-8"
+      >
+        {{ $t('actions.models.tag.select') }}
+      </div>
+      <q-chip
+        v-for="tag in modelValue"
+        :key="tag"
+        dense
+        color="accent"
+        text-color="white"
+        removable
+        icon-remove="fa-solid fa-close"
+        :data-cy="`chip_${tag}`"
+        @remove="unselectTag(tag)"
+      >
+        {{ tag }}
+      </q-chip>
+    </template>
+    <template #option="{ itemProps, opt, selected, toggleOption }">
+      <q-item v-bind="itemProps">
+        <q-item-section side>
+          <q-checkbox
+            color="accent"
+            :model-value="selected"
```

Comment:
can you put `:model-value` first please ?

Reply:
done